client/src/components/musicPlayer.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import $ from 'jquery';
+import songs from '../../sampleData.js'
+
+class VideoPlayer extends React.Component {
+
+	constructor(props) {
+		super(props)
+		this.state = {
+			currentSong: 1
+		}
+	}
+
+	playButtonHandler() {
+		// plays a static song
+	}
+
+
+	artistClickHandler() {
```

Comment:
might not want to make links at all. maybe just have the link change upon hover/click